blas/unit_test/Test_Blas1_axpby_unification.hpp

```diff
@@ -98,7 +98,10 @@ void impl_test_axpby_unification_compare(
     Test::getRandomBounds(max_val, randStart, randEnd);
     Kokkos::fill_random(x.d_view, rand_pool, randStart, randEnd);
   }
-  Kokkos::deep_copy(x.h_view, x.d_view);
+#ifdef HAVE_KOKKOSKERNELS_DEBUG
+  std::cout << "kdc a-001" << std::endl;
```

Comment:
Is that supposed to mean something?

Reply:
I just wanted to save the idea for eventual future needs. 'kdc' = kokkos::deep_copy. I added printout before every deep_copy in the file (around 130 of them). I removed all remaining printouts with kdc (there were 8 only).